base_delivery_carrier_label/tests/test_manifest_wizard.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2017 Angel Moya (PESOL)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+from openerp import fields, exceptions
+
+
+class ManifestWizardCase(TransactionCase):
+    def setUp(self, *args, **kwargs):
```

Comment:
`setUp` doesn't take other argument than `self` you can remove `*args, **kwargs`